src/multiparser.js

```diff
@@ -159,6 +163,62 @@ function fromHtmlParser2(path, options) {
   }
 }
 
+function wrapCssTemplate(quasisDoc, expressionDocs) {
+  const allReplaced = replacePlaceholders(quasisDoc, expressionDocs);
+  if (!allReplaced) {
+    throw new Error("Couldn't insert all the expressions");
```

Comment:
Ok, but this won't be shown to users, it's there just in case a placeholder was parsed in an odd way and we couldn't find it. Throwing the error aborts the formatting of the template literal but the rest of the file should be formatted normally.  
Maybe it should be an assert.